crates/trippy/src/lib.rs

```diff
@@ -0,0 +1,2 @@
+#![allow(rustdoc::broken_intra_doc_links, rustdoc::bare_urls)]
+#![doc = include_str!("../../../README.md")]
```

Comment:
I don't think this is needed, strictly speaking, as this is a binary crate.  Adding this will allow it to appear in docs.rs however.